docs/reference/contrib/modeladmin/primer.rst

```diff
@@ -142,15 +142,20 @@ Overriding templates
 --------------------
 
 For all modeladmin views, Wagtail looks for templates in the following folders
-within your project, before resorting to the defaults:
+within your project before resorting to the defaults:
 
 1. ``/modeladmin/app-name/model-name/``
 2. ``/modeladmin/app-name/``
 3. ``/modeladmin/``
 
-So, to override the template used by ``IndexView`` for example, you'd create a
-new ``index.html`` template and put it in one of those locations.  For example,
-if you wanted to do this for an ``ArticlePage`` model in a ``news`` app, you'd
+As when you `customise the Wagtail admin templates <https://docs.wagtail.io/en/v2.4/advanced_topics/customisation/admin_templates.html?highlight=wagtailadmin#customising-admin-templates>`_, you can create a ``/templates/`` folder within one of your installed apps in order to place the ``/modeladmin/`` folder.  If, for example, you create a ``dashboard`` app for the purpose of containing all logic related to customising the Wagtail admin, then you could set up the following overall folder structure:
+
+-   ``/dashboard/templates/wagtailadmin``
```

Comment:
I recommend removing this wagtailadmin reference, while it appears to be used as an analogy of the folder structure it seems to imply that `modeladmin` templates can also sit in `wagtailadmin`, which will not work (as far as I know :) )

Reply:
I agree, but more because overriding some base templates for the entire project is conceptually very different - The `dashboard` app approach makes sense there. 9 times out of 10, I'd guess that most `modeladmin` template overrides are app or model specific, in which case, they are better off being added to app-specific template folders.

Reply:
Thank you both for your comments.  In proposing this change, I was trying to write in the same manner that the existing Wagtail documentation is written, i.e. Wagtail assumes a certain level of Django knowledge and so doesn't try to re-explain things that are found in the Django docs.  What prompted me to propose this change was that I spent longer than I should have, after following the Wagtail docs, in trying to set up a `modeladmin` customization - the docs said:

> For all `modeladmin` views, Wagtail looks for templates in the following folders within your project, ...

Not thinking (but following the docs), I just placed a `modeladmin` folder in the project, and of course Wagtail couldn't find it.  It was left unsaid in the Wagtail docs that this folder needed to be within a recognized templates folder.  In my proposed change I stated 

> As when you customise the Wagtail admin templates, you can...

and linked to the customisation of the admin templates because I thought it provided a useful cross-reference.  Perhaps, then, just a note about `modeladmin` needing to be in a recognized templates folder would be sufficient, and I definitely agree with the point that `modeladmin` customisation would more likely be done within individual apps than the way I've shown it.

Two questions then:
- If I simply removed the following from this proposed change, and let the rest, would this suffice:
> If, for example, you create a dashboard app for the purpose of containing all logic related to customizing the Wagtail admin, then you could set up the following overall folder structure:
    /dashboard/templates/wagtailadmin
    /dashboard/templates/modeladmin
- Would you be OK with letting the link to customizing the Wagtail admin in as a useful cross-reference, or would you rather see that removed?

Reply:
Thanks @DanAtShenTech.

> In proposing this change, I was trying to write in the same manner that the existing Wagtail documentation is written, i.e. Wagtail assumes a certain level of Django knowledge and so doesn't try to re-explain things that are found in the Django docs

This is a good point, and worth bearing in mind here.

> If I simply removed the following from this proposed change, and let the rest, would this suffice?

If you could try just tweaking the existing content first, to better show that templates should live in `templates/` folders? I think that might possibly be enough to clear up the confusion here.

> Would you be OK with letting the link to customizing the Wagtail admin in as a useful cross-reference, or would you rather see that removed?

I would say let's keep things simple for now and leave the link out. If we can rejig the existing `ArticlePage` override example to show override templates being added to the `news` app folder, I think we'll be about there. I think most of the information on that page is very `wagtailadmin` / branding specific, so isn't actually that relevant (even if it the subject matter is similar).

Reply:
no further comments from me, all makes sense. Thanks @DanAtShenTech  for wanting to make this better, and yeah when I first read the docs the other day I also assumed the templates would go in a 'modelAdmin' folder not a 'templates/modelAdmin' one so I think it is very fair that others will get confused by this and it needs improvements.